### PR TITLE
Feature/migrate mui

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Move the icons-mui repo into icons repo

### CODE REVIEW

**Comment by joebochill** [comment]
If you add new scripts, you need to `chmod 777` them so that they are executable by circleCI:
image

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,31 +7,39 @@ jobs:
...
+                  command: cd mui && yarn test
             - persist_to_workspace:
                   root: .
                   paths:
                       - iconfont
                       - svg
                       - rn-vector
+                      - mui/dist
 
     publish:
```
Comment (joebochill) [inline .circleci/config.yml:44]: let's add the automatic tagging while we are in here

File: mui/.prettierignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+icons/*
```
Comment (joebochill) [inline mui/.prettierignore:1]: this should be /dist

File: .gitignore [hunk 3]
```diff
@@ -26,3 +26,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 /.idea/
+
+mui/icons
```
Comment (joebochill) [inline .gitignore:30]: this should be mui/dist

File: mui/PUBLISHING.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+This package is published to NPM automatically by CircleCI when code is merged into the `dev` or `master` branches. To publish a new version, simply update the version in `package.json` and merge your code into the appropriate branch.
+- The `dev` branch will publish versions marked as `alpha` or `beta`.
+- The `master` branch will publish any version (`alpha`, `beta`, or `latest`).
+  In both cases, the code will only be published if the version number differs from the current version published under the respective dist tag.
+
+## Manually Publishing
+
+If you need to publish a package manually, you can run the following commands from the root folder:
+
+From the root:
```
Comment (joebochill) [inline mui/PUBLISHING.md:14]: This is redundant with the sentence above

File: mui/scripts/linkComponents.sh [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+mkdir -p "./demos/showcase/node_modules/@pxblue/icons-mui"
+# rm -rf "./demos/storybook/node_modules/@pxblue/icons-mui"
+# mkdir -p "./demos/storybook/node_modules/@pxblue/icons-mui"
+echo -e "${GREEN}Done${NC}"
+
+echo -en "${BLUE}Copying build output into node_modules...${NC}";
+# cp -r ./icons/package.json ./demos/showcase/node_modules/@pxblue/icons-mui/package.json
+cp -r ./icons/. ./demos/showcase/node_modules/@pxblue/icons-mui
+# cp -r ./components/package.json ./demos/storybook/node_modules/@pxblue/icons-mui/package.json
+# cp -r ./dist/. ./demos/storybook/node_modules/@pxblue/icons-mui
```
Comment (joebochill) [inline mui/scripts/linkComponents.sh:26]: I think the first three lines here are redundant (and pulling from the wrong folder). The last linecopies everything from the dist folder, which should be everything we need.

File: .circleci/config.yml (hunk 1 above)
Comment (emclaug2) [reply .circleci/config.yml:44]: Just added this now.

File: mui/.prettierignore (hunk 2 above)
Comment (emclaug2) [reply mui/.prettierignore:1]: Done

File: mui/PUBLISHING.md (hunk 4 above)
Comment (emclaug2) [reply mui/PUBLISHING.md:14]: Removed the extra `for the root`

File: mui/scripts/linkComponents.sh (hunk 5 above)
Comment (emclaug2) [reply mui/scripts/linkComponents.sh:26]: I'm not sure how, when, if this linking script is used anymore but this should be fixed now.

File: .circleci/config.yml [hunk 6]
```diff
@@ -51,9 +59,33 @@ jobs:
...
+      docker:
+        - image: circleci/node:14.9-browsers
+      steps:
+        - checkout
+        - attach_workspace:
+            at: .
+        - gh/setup
+        - run:
+            name: Publish @pxblue/icons
+            command: yarn publish:package -b $CIRCLE_BRANCH
```
Comment (joebochill) [inline .circleci/config.yml:79]: This does a publish instead of a tag

Comment (joebochill) [reply .circleci/config.yml:79]: The other three descriptions say 'publish' but are performing a tag.

File: .gitignore (hunk 3 above)
Comment (joebochill) [reply .gitignore:30]: still says mui/icons

File: mui/scripts/linkComponents.sh [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+GRAY='\033[1;30m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}Building components...${NC}"
+yarn build
+
+echo -en "${BLUE}Creating new folder in node_modules...${NC}"
+rm -rf "./demos/showcase/node_modules/@pxblue/icons-mui"
+mkdir -p "./demos/showcase/node_modules/@pxblue/icons-mui"
+# rm -rf "./demos/storybook/node_modules/@pxblue/icons-mui"
```
Comment (joebochill) [inline mui/scripts/linkComponents.sh:18]: Can we remove all of the 'storybook' comments in here...there is no storybook demo

File: mui/scripts/linkComponents.sh (hunk 5 above)
Comment (joebochill) [reply mui/scripts/linkComponents.sh]: It's used to link the components up to the demo. I think this file was probably originally copied from the component library which has multiple demo projects.

File: mui/scripts/linkComponents.sh (hunk 7 above)
Comment (emclaug2) [reply mui/scripts/linkComponents.sh]: These comments are removed now

File: .circleci/config.yml (hunk 6 above)
Comment (emclaug2) [reply .circleci/config.yml]: Ah good catch, these are updated and fixed

File: .gitignore (hunk 3 above)
Comment (emclaug2) [reply .gitignore]: It's now mui/dist

**Review by joebochill — APPROVED** [review]

**Pull request merged**